diff --git a/es/inversify.js b/es/inversify.js
index 67a53dc014ea54764b4774d57bfe4c00ad5dcfb8..b784fb1087a1af0eac1b06070df6b28e3e8e5a39 100644
--- a/es/inversify.js
+++ b/es/inversify.js
@@ -23,3 +23,4 @@ export { decorate } from './annotation/decorator_utils';
 export { traverseAncerstors, taggedConstraint, namedConstraint, typeConstraint } from './syntax/constraint_helpers';
 export { getServiceIdentifierAsString } from './utils/serialization';
 export { multiBindToService } from './utils/binding_utils';
+export { getBaseClassDependencyCount, getDependencies, getFunctionName } from './planning/reflection_utils';
diff --git a/lib/inversify.d.ts b/lib/inversify.d.ts
index 283998db08dd810823e14e3fa2eb3e7e726b8c83..d0696e1b7bd247948891cd8427f617f5daceafe2 100644
--- a/lib/inversify.d.ts
+++ b/lib/inversify.d.ts
@@ -23,3 +23,4 @@ export { decorate } from './annotation/decorator_utils';
 export { traverseAncerstors, taggedConstraint, namedConstraint, typeConstraint } from './syntax/constraint_helpers';
 export { getServiceIdentifierAsString } from './utils/serialization';
 export { multiBindToService } from './utils/binding_utils';
+export { getBaseClassDependencyCount, getDependencies, getFunctionName } from './planning/reflection_utils';
